gotenna_packet.py

```diff
@@ -251,3 +251,70 @@ def decode_tlvs(bytes):
         value = bytes[2:2+length]
         bytes = bytes[2+length:]
         decode_tlv(tag, value)
+
+
+def process_data_frag(bytes, fragIDX):
+    """
+    Process a fragment of a data PDU
+    Unfortunately, since we don't channel-hop. we can't do actual
```

Comment:
gr-tenna receives all channels simultaneously (by intentionally aliasing them all into a single channel), so channel hopping is not a problem.